### ISSUE TITLE
Review Reken.dev and supporting documents

### ISSUE THREAD

**Issue by hbroek** [hbroek/reken#19]
The Reken.dev website and various documents need a 2nd set of eyes to look for typos, grammar etc.  English being my 2nd language is my excuse.

**Comment by hbroek** [comment]
@freshgum-bubbles, perhaps you want to give this a shot.

**Comment by freshgum-bubbles** [comment]
> @freshgum-bubbles, perhaps you want to give this a shot.

Happy to give it a look :-)

**Comment by hbroek** [comment]
@freshgum-bubbles, yes, please. I just added a new attribute data-ref and just focus on the new functionality
